docs/core/reference/cored.md

```diff
@@ -0,0 +1,95 @@
+# cored Command
+
+Command `cored` provides the Chain Core daemon and API server.
+
+`cored` is included with all official releases of Chain Core
+Developer Edition, and is run by default when starting Mac, Windows and Docker.
+
+If you plan to make changes to the internals of Chain Core, you can find
+build instructions in our [GitHub readme](https://github.com/chain/chain/blob/main/Readme.md#building-from-source);
+if you want to _use_ chain core, please download one of our official releases.
+
+### Build tags
+
+There are four build tags that change the behavior of the resulting binary:
+
+  - `reset`: allows the core database to be reset through the api
+  - `loopback_auth`: allows unauthenticated requests on the loopback device (localhost)
+  - `no_mockhsm`: disables the MockHSM provided for development
+  - `plain_http`: allows plain HTTP requests
+
+Chain Core Developer Edition is built with `reset`, `loopback_auth` and
+`plain_http` enabled.
+
+## Flags
+
+* **-version**: Print version and build information
+
+## Environment Variables
+
+Many functions of Chain Core can be controlled by the presence of environment
+variables. For example, to start a Chain Core that listens on a different port:
+
+```
+LISTEN=:9876 cored
+```
+
+### Basic Functions
+
+The following variables control the location of Chain Core's data stores,
+and its routable address.
+
+* **CHAIN_CORE_HOME**: Path to Chain Core data directory on local file system,
+defaults to `$HOME/.chaincore`.
+
+* **DATABASE_URL**: URL of the Postgres database, defaults
+to `postgres:///core?sslmode=disable`.
+
+* **LISTEN**: Address the Chain Core server will listen on, defaults
+to `:1999`. In a multi process configuration, this must be a
+reachable, routable address.
+
+### Extended Functionality
+
+The following variables allow for more fine-grained operation control of
+Chain Core, such as log rotation, rate limits, and clustering.
+
+* **ROOT_CA_CERTS**: Path to file containing a set of PEM-encoded concatenated
```

Comment:
I changed the section titles from "XYZ Auth" to "XYZ authentication". So the hashlinks will need to get updated too. Sorry 😬